src/main/java/apoc/number/ArabicRoman.java

```diff
@@ -0,0 +1,62 @@
+package apoc.number;
+
+
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.UserFunction;
+
+public class ArabicRoman {
+
+    @UserFunction
+    @Description("apoc.number.romanToArabic(romanNumber)  | convert roman numbers to arabic")
+    public Number romanToArabic(final @Name("romanNumber") String value) {
+        return toArabic(value);
+    }
+
+    @UserFunction
+    @Description("apoc.number.arabicToRoman(number)  | convert arabic numbers to roman")
+    public String arabicToRoman(final @Name("number") Object value) {
+        Number number = validateNumberParam(value);
+        if (number == null) return null;
+        return getRoman(number.intValue());
+    }
+
+    private Number validateNumberParam(Object number) {
+        return number instanceof Number ? (Number) number : null;
+    }
+
+    private static int toArabic(String number) {
```

Comment:
Wouldn't it make more sense to move all the static methods into the Enum as instance methods where possible ?
And have them be self-aware? I.e. if number.startsWith(this.name()) return ....
etc.